install/helm/agones/templates/crds/_gameserverspecvalidation.yaml

```
@@ -81,6 +81,7 @@ properties:
               enum:
```

Please update the description. Something like:
```suggestion
              title: Protocol being used. Defaults to UDP. TCP and TCPUDP are other options.
              type: string
              enum:
```